parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -509,10 +509,11 @@ configGPUDrivers() {
   retrycmd_if_failure 120 5 25 mount -t overlay -o lowerdir=/usr/lib/x86_64-linux-gnu,upperdir=${GPU_DEST}/lib64,workdir=${GPU_DEST}/overlay-workdir none /usr/lib/x86_64-linux-gnu || exit {{GetCSEErrorCode "ERR_GPU_DRIVERS_CONFIG"}}
   retrycmd_if_failure 3 1 600 sh $GPU_DEST/nvidia-drivers-$GPU_DV --silent --accept-license --no-drm --dkms --utility-prefix="${GPU_DEST}" --opengl-prefix="${GPU_DEST}" || exit {{GetCSEErrorCode "ERR_GPU_DRIVERS_START_FAIL"}}
   echo "${GPU_DEST}/lib64" >/etc/ld.so.conf.d/nvidia.conf
+  cp "${GPU_DEST}/bin" /usr/bin
```

Comment:
What is special about `/usr/bin` as opposed to `/usr/local/nvidia/bin`?

Reply:
`nvidia-container-runtime` and `nvidia-container-cli` bins are located in `/usr/bin`.

It seems it will only search nvidia utility/compute bins in the same folder and then mounted them accordingly into container. Because I've tried to add `/usr/local/nvidia` into system path, and it does not work as expected.

I also opened an issue https://github.com/NVIDIA/nvidia-docker/issues/1226, but there's no response yet. 

So this's a quick fix for this issue.